softwerkskammer/lib/activities/activitiesService.js

```diff
@@ -130,14 +123,14 @@ module.exports = {
     });
   },
 
-  addToWaitinglist: function (memberId, activityUrl, resourceName, moment, callback) {
-    var self = this;
-    activitystore.getActivity(activityUrl, function (err, activity) {
+  addToWaitinglist: function addToWaitinglist(memberId, activityUrl, resourceName, moment, callback) {
+    const self = this;
```

Comment:
Ich behaupte jetzt mal (orakel orakel) dass wir hier gar kein self mehr benötigen, weil die lambdas das this nicht überschreiben. Ich vermute also, man würde hier prima mit this.blabla arbeiten können.
(Kannst Du ausprobieren, wenn Du magst, musst Du aber nicht.)